frontend/src/libs/components/stepper/project-name-step/project-name-step.tsx

```diff
@@ -0,0 +1,54 @@
+import { joiResolver } from '@hookform/resolvers/joi';
+import { useForm } from 'react-hook-form';
```

Comment:
Shouldn't be used custom useAppForm hook? 
```suggestion
import { useAppForm } from '~/libs/hooks/hooks.js';
```

Reply:
im added atribute mode, so used useForm

Reply:
you can add new properties to useAppForm